spec.bs

```diff
@@ -391,6 +422,8 @@ below and in [[#index-defined-elsewhere]].
     away from its Payment Request origins. For developers, this should
     improve feature detection, invocation, and other aspects of the
     API.</div>
+    
+    Q: Is the suggestion to create SPC specific methods, or roll everything into WebAuthn, with no custom-UI/logic in the browser agent?
```

Comment:
I'll leave the question of API shape for now, but note that we (Chrome) currently expect to always present payment-specific UX for an SPC authentication, no matter the API shape to call into it.

My personal understanding is that this NOTE only seeks to change the JavaScript code used to call SPC but none of the current behavior or UX would change.